apps/expo/package.json

```diff
@@ -194,6 +195,7 @@
     "react-native-level-fs": "^3.0.0",
     "react-native-localhost": "^1.0.0",
     "react-native-mmkv": "2.5.1",
+    "react-native-modal": "^13.0.1",
```

Comment:
are we using react-native-modal anywhere?

Reply:
Yes, hCaptcha has this dependency